src/ng-select/items-list.ts

```diff
@@ -188,4 +188,8 @@ export class ItemsList {
     private get _lastSelectedItem() {
         return this._selected[this._selected.length - 1];
     }
+
+    isMaximumNumberOfSelectedItemsReached(): boolean {
```

Comment:
Move this public method before all private methods.

Reply:
Also I would suggest renaming this method to less verbose, eg.: `maxItemsSelected` or similar